### DISCUSSION TITLE
Acceptible data file format(s)

### DISCUSSION THREAD

**Discussion by tappanbr** [AUTODIAL/AutoEIS#18] [Q&A]
Hello! Thank you for making AutoEis. When trying to use it, I was not able to find any documentation on how the imput.txt file should be formatted. It would be great if you could provide some detail on that. Is it simply a text file with three columns, one for freq, one for imag real impedance and one for imag impedance? When exporting EIS files from EC-lab, for example, headers are included in the text file, and I wasn't sure if these should be included or not.

Thank you

**Comment by ma-sadeghi** [comment]
Hi @tappanbr! Sorry for the late reply. We're in the process of restructuring the package to make it easier to use/install. Hopefully, we'll publish a new release on PyPI soon (timeline: 1-10 days maybe?). Stay tuned, and thanks for your patience!

**Reply by ma-sadeghi** [reply]
@tappanbr Sorry for the delay, we've released v0.0.18, which involves lots of API changes, refactors, and performance boosts! Please give it a try and let us know what you think!

Re file format: We currently don't have any data parsers, for parsing different data formats, try [pyimpspec](https://vyrjana.github.io/pyimpspec/guide_data.html). I haven't tried it myself, but seems a stable package.

As for inputs for AutoEIS, as long as you have two arrays, impedance and frequency as numpy arrays, you can pass them to AutoEIS. Please refer to our [documentation](https://autodial.github.io/AutoEIS/), and of course feel free to open a new issue if you have further questions.
